src/commands/instance/spawn.ts

```diff
@@ -108,7 +108,7 @@ export default class InstanceSpawn extends Kommand {
 
     const { flags: userFlags } = this.parse(InstanceSpawn)
     const portIndex = await this.findAvailablePort()
-    const dockerComposeFileName = `/tmp/kuzzle-stack-${portIndex}.yml`
+    const docoFilename = `/tmp/kuzzle-stack-${portIndex}.yml`
```

Comment:
`docoFilename` or `docFilename` ?

Reply:
nah it's for `DOcker COmpose filename"

I preferred the previous form, I think it's more meaningful that way, but that's nitpicking.

Reply:
I plead guilty, I renamed it to avoid wrapping a line bellow :sweat_smile: 
I prefer to be very explicit also but `doco` was already used among this file so I keep this shortcut

Reply:
Didn't knew that it could be called that way !